lib/grammar.js

```diff
@@ -391,6 +391,31 @@ var grammar = module.exports = {
       reg: /^max-message-size:(\d+)$/,
       format: 'max-message-size:%s'
     },
+    {
+      // RFC7273
+      // a=ts-refclk:ptp=IEEE1588-2008:39-A7-94-FF-FE-07-CB-D0:37
+      name:'timeStampRefClk',
+      reg: /^ts-refclk:([^\s=]*)(?:=(\S*))?/,
+      names: ['clksrc', 'config'],
+      format: function (o) {
+        return 'ts-refclk:%s' + (o.config !== undefined ? '=%s' : '');
+      }
+    },
+    {
+      // RFC7273
+      // a=mediaclk:direct=963214424
+      name:'mediaClk',
+      reg: /^mediaclk:(?:id=(\S*))? *([^\s=]*)(?:=(\S*))?(?: *rate=(\d+)\/(\d+))?/,
```

Comment:
This is a very involved regex. It would be good to have some test cases for it because as it stands it's very easy to accidentally break it.